_localtest/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.jcraft:jsch:0.1.53'
-        classpath group: "jaci.gradle", name: "EmbeddedTools", version: "2018.11.05"
+        classpath group: "jaci.gradle", name: "EmbeddedTools", version: "2018.12.222"
```

Comment:
?

Reply:
I didn't know when this would be merged, so I just picked a random number. This is whats in the main build.gradle too.